### PR TITLE
Add migration steps for Java and Android v5

### PR DESCRIPTION
**Pull request by marandaneto**

### CODE REVIEW

File: src/platforms/android/migration.mdx [hunk 1]
```diff
@@ -92,11 +92,15 @@ You may remove `android:extractNativeLibs="true"` meta-data in the `AndroidManif
 
 `Sentry#startTransaction` by default does not bind created transaction to the scope. To start transaction with binding to the scope, use one of the new overloaded `startTransaction` methods taking `bindToScope` parameter and set it to `true`. Bound transaction can be retrieved with `Sentry#getSpan`.
 
+All SDK methods have been annotated with [JetBrains Annotations](https://github.com/JetBrains/java-annotations): `@Nullable` and `@NotNull`. Kotlin compiler respects these annotations and as a result, in Kotlin code, some fields that were recognized as not-null, are now nullable, and the other way around.
+
+`SentryBaseEvent#getOriginThrowable` has been deprecated in favor of `SentryBaseEvent#getThrowable`, `SentryBaseEvent#getThrowable` now returns the unwrapped throwable.
```
Comment (imatwawana) [inline src/platforms/android/migration.mdx:97]:
```suggestion
All SDK methods have been annotated with [JetBrains Annotations](https://github.com/JetBrains/java-annotations): `@Nullable` and `@NotNull`. Kotlin compiler respects these annotations, so in Kotlin code, some fields that were recognized as not-null, are now nullable, and the other way around.

`SentryBaseEvent#getOriginThrowable` has been deprecated in favor of `SentryBaseEvent#getThrowable`, and `SentryBaseEvent#getThrowable` now returns the unwrapped throwable.
```

File: src/platforms/java/migration.mdx [hunk 2]
```diff
@@ -10,6 +10,12 @@ description: "Learn about migrating from io.sentry:sentry 1.x to io.sentry:sentr
 
 All SDK methods have been annotated with [JetBrains Annotations](https://github.com/JetBrains/java-annotations): `@Nullable` and `@NotNull`. Kotlin compiler respects these annotations and as a result, in Kotlin code, some fields that were recognized as not-null, are now nullable, and the other way around.
 
+`SentryBaseEvent#getOriginThrowable` has been deprecated in favor of `SentryBaseEvent#getThrowable`, `SentryBaseEvent#getThrowable` now returns the unwrapped throwable.
```
Comment (imatwawana) [inline src/platforms/java/migration.mdx:13]:
```suggestion
`SentryBaseEvent#getOriginThrowable` has been deprecated in favor of `SentryBaseEvent#getThrowable`, and `SentryBaseEvent#getThrowable` now returns the unwrapped throwable.
```

**Review by imatwawana — APPROVED** [review]
Minor wording edits

**Comment by marandaneto** [comment]
> Minor wording edits

thanks

**Pull request merged**